src/main/java/com/keniu/controllers/BookController.java

```diff
@@ -71,6 +79,7 @@ public BookDto save(@Valid @RequestBody CreateBookRequestDto createBookRequestDt
      * a {@link CreateBookRequestDto} containing the new details of the book
      * @return a {@link BookDto} representing the updated book with the new details
      */
+    @Operation(summary = "Update or crate book")
```

Comment:
This method is for updating, you have a separate method for creating.

Reply:
And, if it's not too much trouble, could you please pass this on to your colleague @Elena-Bruyako as well:
The PUT method requests that the state of the target resource be
   **### created** or replaced with the state defined by the representation
   enclosed in the request message payload.
   https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.4
  Thank you

Reply:
Yes, it's correct to use PUT to create a resource with a known URI.  Do you actually intend to provide an ID for the entity upon creation? Unless you do, this description may be misleading, and the Javadoc currently suggests it’s only for updating an existing resource.

Keep in mind that the ID for this table is set to auto-increment, so manually setting IDs seems unnecessary. Consider how the front end could reliably determine a non-conflicting ID to assign, and why it would even need to. This approach might make sense if a unique, meaningful identifier (like an email) were used, but here, it doesn’t add value. In scenarios like this, POST is generally the better choice for creating new resources.

Reply:
Yes, updating by id due to incrementing is not the best idea here, but I didn’t come up with this task...
Of course, we can create objects using GET methods, we can do anything. But there are HTTP and RESTful API standards These standards tell us that the PUT method should update the entity if it exists and **### create** it if it does not. This is what happens in my method, and the appropriate description has been added. Do we want to update by email? I’m in favor. But the task specifies updates by id...